src/main/java/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;
+
+import db.FruitStorage;
+import java.util.Map;
+import service.ReportGenerator;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReportFromDB() {
+        StringBuilder stringBuilder = new StringBuilder()
```

Comment:
do not use such naming(`stringBuilder`), how to guess what it stores?
```suggestion
        StringBuilder report = new StringBuilder()
```